### ISSUE TITLE
Gran Turismo 3 & 4 reflections are messed up when using Direct3D 12

### ISSUE THREAD

**Issue by antoniodesousa** [TheRhysWyrill/XBSX2#17]
## Description

The reflections on the cars are messed up when using Direct3D 12. They render correctly when using Direct3D 11 or software mode.

### Expected behavior

The reflections on the cars render correctly when using Direct3D 12.

### Actual behavior

The reflections on the cars are messed up when using Direct3D 12.

### Steps to reproduce the bug

1. Launch the game using Direct3D 12
2. Start a race in any mode
3. The reflections on the cars are messed up

### Console
[Xbox Series S]

**Comment by antoniodesousa** [comment]
Sadly, since the latest releases there is no way to fix the cars flickering issue... using D3D 11 or 12 makes no difference. The problem is that the option "Accurate Destination Alpha Test" has been removed in the latest builds, which was need it to fix the flickering issue by turning this option off and using D3D 11.

I did some investigation about it and apparently the PCSX2 team removed "Accurate DATE" in this PR in favor "PrimID DATE":
https://github.com/PCSX2/pcsx2/pull/7003

The weird thing is that I don't have this issue when running PCSX2 on my laptop (either D3D 11 or 12). Maybe, if we could have an option to disable this "PrimID DATE" in the GUI, we could fix the flickering issue in GT 3 & 4 on XBox. It's just a hunch since disabling the "Accurate DATE" did fix the flickering before.
